	<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/bischdog" target="_blank">bischdog</a>
			<div class="markdown"><p>This should be at the top. It contains the best answer with the most relevant information. (Studied in and worked in bioinformatics for 10+ years.)</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/420Microbiologist" target="_blank">420Microbiologist</a>
			<div class="markdown"><p>Except that it doesn’t talk about how you could identify the origin of the leaf even if it’s a clonal colony by using methyl-seq and acetyl-seq data. Just like you could to distinguish human twins from each other!</p></div>		</li>
					</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/EwanPorteous" target="_blank">EwanPorteous</a>
			<div class="markdown"><p>One of the main Forensics Labs used by the police in the UK cannot do it, because as you say they there, has not been any research into tree DNA for profiling purposes.</p>
<p>They cannot do dogs (or any pets) either for that matter.</p>
<p>Source: Enquiries during Police investigations.</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/CrateDane" target="_blank">CrateDane</a>
			<div class="markdown"><p>Then again, sequencing has gotten so cheap and routine that it wouldn't be insurmountable to just do it from scratch if necessary.</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/flabby_kat" target="_blank">flabby_kat</a>
			<div class="markdown"><p>Sort of. Sequencing results are returned as a very long list of short DNA fragments. In species that have a reference genome like humans it's easy enough to turn this information into something usable because you can take each sequenced snippet of DNA and say, &quot;this piece matches this one part of chromosome 3&quot; or whatever and you can just take the sequenced pieces and put them where they belong. When there is no reference genome (like in trees) you have to do what's called a <em>de novo</em> assembly. This is much harder because you have take the pieces and put them together with no information on what your final product should look like. In both cases you have a shredded book that you have to put back together, the difference is, if you have a reference genome, you at least know what the book is supposed to say. Assembling genomes <em>de novo</em> ends up being very expensive because a lot more time, energy, computation, and data is required.</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/yerfukkinbaws" target="_blank">yerfukkinbaws</a>
			<div class="markdown"><p>There's no need to do any of that, though. Much, much simpler methods like AFLPs and RAPD markers are more than capable of differentiating individuals and even clonal ramets and they have a long history of being used for purposes exactly like this. Just because we <em>can</em> do whole genome sequencing and get a boatload of data, doesn't mean it's the best option for every particular question. In genetics: work smarter, not harder.</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/CrateDane" target="_blank">CrateDane</a>
			<div class="markdown"><p>I know. Reads, contigs, scaffold etc. It's just that you can sequence so much faster that getting good depth isn't ludicrously expensive. And you don't necessarily have to get all the tricky repetitive parts of the genome properly sequenced for the particular purpose here.</p>
<p>People are even starting to talk about it not being worth storing sequences electronically anymore, because it's so cheap to sequence. Probably a little premature until long-read methods (nanopore etc.) improve, but the fact that it's even being talked about...</p></div>		</li>
					</ul>
		</ul>
		</ul>
	