	<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/Cadel13" target="_blank">Cadel13</a>
			<div class="markdown"><p>How would you be able to reliably tell if the new, assembled genome is of actual bacteria instead of just some random sequence that happens to make some sense?</p>
<p>Edit: Thank you all for the answers, it was extremely interesting to read.</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/Kandiru" target="_blank">Kandiru</a>
			<div class="markdown"><p>When you assembly a genome, you find overlapping reads which join together. The depth at any point in the assembly is the number of reads that support that location existing.</p>
<p>You can use to calculate the chance of finding that in random sequences. It's normally going to be essential zero for large depth and assembly length.</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/TransmutedHydrogen" target="_blank">TransmutedHydrogen</a>
			<div class="markdown"><p>Good answer. </p>
<p>There are also sections of the genome that are refractory to this process as they are repetitive, this unfortunately comprises almost half the human and mouse genomes etc.</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/davorzdralo" target="_blank">davorzdralo</a>
			<div class="markdown"><p>That explains how we know that it's an actual sequence, not how we know it's a bacteria.</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/mlmayo" target="_blank">mlmayo</a>
			<div class="markdown"><p>What are random sequences?  To my understanding, the 4 bases are not present in equal abundance in most organisms, so youâ€™d need to use different distributions in your random calculations.</p></div>		</li>
					</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/drkirienko" target="_blank">drkirienko</a>
			<div class="markdown"><p>There are a number of ways, depending upon how interested you are in it. The first way is to check if it assembles to something that looks like a whole genome. For bacteria, for example, you check to see if it seems to be a whole circle. (Or parts that seem like they would be.) </p>
<p>If it is, you can look for certain sequences and look to see how they're related to known bacteria. That will tell you (sorta) where this falls in the tree of life, and whether that makes sense. (It can also tell you if it is something that is already known. </p>
<p>If it is new, and all checks out, you can start looking to see if it has all of the genes it would need for how you think it lives. I.e., does it have the genes to metabolize sugar? To photosynthesize? Metabolize methane? etc. This is longer and harder, but can answer more interesting questions.</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/The_camperdave" target="_blank">The_camperdave</a>
			<div class="markdown"><blockquote>
<p>For bacteria, for example, you check to see if it seems to be a whole circle.</p>
</blockquote>
<p>A whole circle?  I thought DNA was just a long strand.</p></div>		</li>
					</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/That_Biology_Guy" target="_blank">That_Biology_Guy</a>
			<div class="markdown"><p>As others say, genomes are assembled by generating enough DNA sequences to cover everything several times over.  In the Boothby study, they had approximately 60x coverage, meaning that on average every part of the genome was read and contained in 60 different DNA fragments.  Based on how much they overlap, these reads are then <a href="https://camo.githubusercontent.com/ec2540fc76dc268f30ba7e54e615584cf97242d6/687474703a2f2f696d6167652e736c696465736861726563646e2e636f6d2f62696f3135336d6963726f6269616c67656e6f6d696373323031322d3132303131353130303933312d70687061707030312f39352f62696f3135332d6d6963726f6269616c2d67656e6f6d6963732d323031322d32352d3732382e6a70673f63623d31333236363234393134" target="_blank">grouped into contigs and scaffolds</a>.  In a perfect world, you would end up with one scaffold per chromosome (since these are physically separate and don't overlap), but in reality this never really happens.</p>
<p>Now with that background aside, the biggest indication that the bacterial DNA was from contaminants is the fact that there were cases where an <em>entire contig</em> consisting of thousands of genes <em>in a row</em> was entirely bacterial.  If the foreign DNA was actually from horizontal gene transfer, we would expect it to be spread out in various parts of the genome.  There is no known mechanism by which a bacterium's full genome can be inserted into a eukaryote's all at once, and even if this could happen it would be broken up by chromosomal rearrangements relatively quickly on an evolutionary time scale.</p>
<p>So the fact that they found entire contigs consisting of all bacterial DNA and no tardigrade DNA means that there were full, intact bacterial chromosomes in the sample, which really only makes sense if they came directly from bacteria.</p></div>		</li>
					</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/softnmushy" target="_blank">softnmushy</a>
			<div class="markdown"><p>Did anyone subsequently sequence a tardigrade genome that did not have that bacterial DNA?</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/That_Biology_Guy" target="_blank">That_Biology_Guy</a>
			<div class="markdown"><p>Yes!  The raw data from the Boothby et al. study were reassembled using some different techniques by <a href="https://www.ncbi.nlm.nih.gov/pubmed/27069789" target="_blank">Delmont and Eren 2016</a>.  Additionally, <a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2002266" target="_blank">Yoshida et al. 2017</a> used a combination of this old data as well as newly generated sequences to make their own assembly of both <em>Hypsibius dujardini</em> and a second species, <em>Ramazzottius varieornatus</em>.  You can access all four of these assemblies from <a href="https://www.ncbi.nlm.nih.gov/assembly/?term=txid42241%5BOrganism%3Aexp%5D" target="_blank">Genbank</a> if you want.</p></div>		</li>
					</ul>
		</ul>
	