	<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/ValidatingUsername" target="_blank">ValidatingUsername</a>
			<div class="markdown"><p>OT</p>
<p>Someone mentioned in another thread that the driving mechanism for epigenetic shifts is caused by methyl groups and it's called methylation, could you please point me in the direction of some accurate information on the topic?</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/Xilon-Diguus" target="_blank">Xilon-Diguus</a>
			<div class="markdown"><p>This is incorrect, or at least incomplete. There are two main forms of methylation in an epigenetic context, <a href="https://en.wikipedia.org/wiki/DNA_methylation" target="_blank">DNA methylation</a> is the covalent addition of a methyl group to cytosine. It can also happen to adenosine but that is a more complicated, less well-studied mechanism. The addition of methyl groups to DNA is usually correlated with compact, inaccessible DNA alongside repressive histone marks. It occurs at a CG context in humans and it maintained when the DNA divides as the original strands will have a methyl mark at the CG site, allowing for the cell to methylate the opposite cytosine. </p>
<p>The second type of methylation is histone tail methylation, usually on lysine residues (K4, K9, K27, and K36 are the big players). These can either have the effect of making the DNA more accessible (Histone 3 Lysine 4 trimethylation or H3K4me3) or more compact (H3K27me3). There are also other possible marks on amino acid residues like acetylation (the other big player here), ubiquitination, and phosphorylation. There marks on histone tails are then read by various proteins in the cell which adjust the state of the DNA packaging. The exact specifics on what mark does what is somewhat conserved across eukaryotes, but there is some variation.</p>
<p>I should also say that histone varients are very important for epigenetic shifts. Histone variants (like H2A.Z) are just different versions of histone subunits that are deposited at different areas in the genome and coorelate with chromatin state. </p>
<p>Unfortunately, I don't just have one resource for methylation as it is a pretty broad topic, is there any particular aspect of epigenetic regulation you are interested in?</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/ValidatingUsername" target="_blank">ValidatingUsername</a>
			<div class="markdown"><p>You summarized what I was looking for quite succinctly.</p>
<p>Thank you very much.</p></div>		</li>
					</ul>
		</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/shreyaspandit" target="_blank">shreyaspandit</a>
			<div class="markdown"><p>Here is the study i was referencing to : <a href="https://www.cnbc.com/2020/05/04/coronavirus-uae-stem-cell-treatment-sees-favorable-outcome.html" target="_blank">https://www.cnbc.com/2020/05/04/coronavirus-uae-stem-cell-treatment-sees-favorable-outcome.html</a></p></div>		</li>
					</ul>
	