	<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/ChamberKeeper" target="_blank">ChamberKeeper</a>
			<div class="markdown"><p>So then transcription factors are what make up the epigenome, yes? </p>
<blockquote>
<p>These sequences are called “regulatory elements” and do not actually code a protein, so they are not considered genes. But don’t call them junk!</p>
</blockquote>
<p>Does it make sense to think of transcription factors as <em>epigenes</em>?</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/mix_feedback_repeat" target="_blank">mix_feedback_repeat</a>
			<div class="markdown"><p>The term &quot;epigenetics&quot; is hotly debated, but in a very general sense, I would say yes, transcription factors and controlling elements are core components of epigenetics. IMO, epigenetics as a field of study is concerned precisely with what you are asking about: the regulation of gene expression and the heritable &quot;program&quot; that directs said regulation. However, some of the other components controlling gene expression complicate the issue immensely, take for instance DNA methylation, which adds a kind of molecular &quot;flag&quot; to the DNA that can prevent a controlling element from being recognized by the transcription factor.</p>
<p>Although the term &quot;epigene&quot; is not really used in genetics, its not a bad way to think of transcription factors.</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/GeneralEi" target="_blank">GeneralEi</a>
			<div class="markdown"><p>&quot;Epiallele&quot; comes up a bit infrequently tho which doesn't exactly help to complicate the matter when trying to explain that &quot;epigenes aren't <em>technically</em> a thing&quot;</p></div>		</li>
					</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/CocktailChemist" target="_blank">CocktailChemist</a>
			<div class="markdown"><p>An important thing to remember is that all of this is stochastic. All of the different components are bouncing around and interacting with each other in different ways, so epigentic changes influence the chance that DNA or protein will bind with another protein or change its conformation. </p>
<p>&#x200B;</p>
<p>Take, for instance, histone methylation and acetylation. These post-translational modification happen on lysine (or arginine, but let's put that aside for the moment) residues on the histone. Lysines have terminal amines, which are usually charged at physiological pH and make for strong interactions with negatively charged DNA. Methylation of those lysines makes them even more likely to be charged and even better binding partners, tightly binding the histones to the DNA in a way that makes them less accessible to transcription factors and polymerases. Acetylation blocks those interactions by making the lysines neutral, so it's harder for DNA to bind to the histones. That results in more loosely packed DNA that is more accessible to transcription factors and polymerases. Importantly, all of these modifications are reversible, so enzymes can put them on or take them off to influence gene expression at different times.</p>
<p>&#x200B;</p>
<p>This is overly simplified, because lysine methylation is also sometimes involved with transcriptional activation and DNA repair, but the basic schema is close enough.</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/bme2023" target="_blank">bme2023</a>
			<div class="markdown"><p>Why does methylation increase lysine's charge? I would expect that both a methyl and an acetyl group would bind to the positively charged amine on lysine, no?</p></div>		</li>
					</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/Prasiatko" target="_blank">Prasiatko</a>
			<div class="markdown"><p>Specifically tracscription factors and their effects on transcription are the <em>transcriptome.</em></p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/mix_feedback_repeat" target="_blank">mix_feedback_repeat</a>
			<div class="markdown"><p>The transcriptome is the total population of RNA expressed at a given time in a given cell type.</p></div>		</li>
					</ul>
		</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/shazzzaa" target="_blank">shazzzaa</a>
			<div class="markdown"><p>^^ and TF’s can function depending on certain metabolic signals. enough energy (what/how much youre eating, how its being broken down etc), amongst other things. exposure to sunlight etc!</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/oss_spy" target="_blank">oss_spy</a>
			<div class="markdown"><p>That's very complicated even in a short answer and I can see why it's such a big challenge</p></div>		</li>
					</ul>
	