	<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/craftmacaro" target="_blank">craftmacaro</a>
			<div class="markdown"><p>How’s it going, I can’t help but notice my research in a specific part of my dissertation overlaps with your research as well as the literature you’ve shared in a lot of ways (I’ve pretty much been assaying and looking for the presence of apoptosis or necroptosis via a number of flow techniques and some liberal usage of MTT and MTS assays.  I study venomous snakes (I work in the field and do extractions) but with a particular emphasis of bioprospecting venom proteins for potential anti-cancer, cancer or cell diagnostic tools, and other applications for medical utility (with great bioactivity, diversity, and specificity comes some shotgun approaches to research sometimes).  I’ve discovered that the snake venom metalloproteases of a particular south American snake that is very understudied is very cytotoxic (most snake venom metalloproteases are pretty much cell killers no matter what line so it’s not a family we expected much specificity and haven’t seen the kind of specificity I’ve found in this species’ venom).  Essentially it’s even more potently cytotoxic than the average for other snakes in its family and other venomous families but unlike those others one of my experimental lines (a375 melanoma) are completely resistant judging by both 24 and 48 hour viability in MTT and MTS assays as well as flow (the morphology does change but the cells survive and even multiply just fine).  I’ve been poking around looking for the most likely reason for this resistance, (anoikis dependency... but it wrecks Colo205 cells which aren’t even a fully adherent cell line) I’ve been looking at necroptosis vs apoptosis but so far it doesn’t seem to be connected (in the cell lines it does kill it is initiating Posphotidyl serine flipping detectable by Annexin V) I’m planning on using PARP cleavage to confirm apoptosis, but as it’s toxic to healthy cells in the same doses it’s not exciting as a direct therapeutic.  </p>
<p>What I was wondering from you is if you’ve encountered much distinguishing A375’s or another melanoma line from most other lines you might be familiar with.  I’ve been thinking neural crest origin and the N Cadherin switch might have a roll but they are pretty much shots in the dark.  Activation of transcription factors is certainly going to be induced by any enzymatic toxins that are clipping extra cellular proteins and domains of membrane bound proteins that happen to be exposed and have the right sequence, and I would certainly love to get a good look at the overall change in which transcription factors are initiated and which are inhibited after exposure (I’d love to see this for any toxin).  That second paper you linked had some interesting apoptosis inducing transcription factors but before heading down that rabbit hole with little to know direction I’d love to hear what sort of assays, antibodies, stains, and techniques you’ve found both applicable and useful without too much individual tweaking between labs and cell lines... because I think we both know how easy something can look in a methods section compared to actually putting it into practice with your own cells.  Anyway, even if your research is way less tangential than I’m hoping I hope you have good luck in all your experiments and may your control populations always behave predictably.</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/fluffyrhinos" target="_blank">fluffyrhinos</a>
			<div class="markdown"><p>Thanks for the question. Your research sounds really cool! I'm quite glad that someone else is doing research involving harvesting venom from snakes, because I certainly would not be cut out for that sort of science! I think my research is less tangential, so while this comment turned out long, unfortunately I think you'll find it contains little useful info. </p>
<p>I've only worked with a couple cell lines (Raw, 3T3, Hela, 293), so I don't think I can help answer anything specific to A375s. My first instinct was to think about the adherence, simply because that's what I picture being cleaved by metalloproteases. Given that, I think looking at the cadherin proteins being expressed, like you mentioned, is a good path to investigate. I really don't know as much about apoptosis as I probably should, otherwise I might have been able to give you more info. Because I think about signaling a lot, I think about apoptosis like a balancing act between pro- and anti- signaling pathways, and it's possible that the reason your A375s survive is because of some factor giving strong anti-apoptosis signals.</p>
<p>Have you done any RNAseq with these cells? Cross referencing the transcriptome data with the transcription factors known to regulate those genes could give you some hits.</p>
<p>My lab uses microscopy to do our measurements, so I can help from that perspective. Unfortunately, antibodies used for immunofluorescence can be pretty unreliable, so that might come down to trial and error. Also, given that they can be pretty expensive, if you're planning on looking for several potential targets, that's probably not the way to do it. </p>
<p>If your lab is set up to do live-cell microscopy, there are a number of quality reporters on addgene that you could transform into your cell lines. Certainly good reporters exist for p53, but probably other transcription factors as well. This isn't something I would recommend if your lab hasn't done it before, but if you have, it could be straight forward and I'd be happy to send examples of reporters I've used in the past.</p>
<p>I know that library prep can be a pain in the ass, but doing a large scale screen in your cell line could reveal some genes acting to keep the cells alive. And in the cells normally killed by the toxin, it could reveal the target of the toxin.</p>
<p>I hope this helped in any small way, but I'm sorry I wasn't able to provide more specific advice. I wish you luck with your dissertation, and lots of luck not getting bitten by snakes!</p></div>		</li>
					</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/CookieKeeperN2" target="_blank">CookieKeeperN2</a>
			<div class="markdown"><p>the way you talk about tf, don't you mean they are proteins? yes they are proteins which are expressed from corresponding genes, but the way you sound you make it as if genes themselves are regulating other gene expressions.</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/fluffyrhinos" target="_blank">fluffyrhinos</a>
			<div class="markdown"><p>Sorry I wasn't clear. You are right, TFs are indeed proteins. When I talk about dynamic activation, I'm not referring to the genes being expressed, I'm referring to the protein itself being activated. How this happens depends on the specific protein. </p>
<p>For example, NF-kB is normally bound by an inhibitory protein that keeps it in the cytoplasm, away from the DNA. Upon activation, this protein is degraded and NF-kB translocates to the nucleus where it can regulate gene expression. p53 on the other hand, gets degraded when it is supposed to be inactive, and that degradation is inhibited when it is supposed to be activated. That way, there are only high levels of p53 when it is supposed to be active.There are other mechanisms as well. </p>
<p>Many transcription factors are regulated in these post-translational ways, because it means they can react faster. NF-kB, which regulates many immune genes, can react in a matter of minutes to a stimulus, which wouldn't be possible if the cell had to express the NF-kB proteins from the DNA when it needs to respond.</p>
<p>I hope this answered your question!</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/CookieKeeperN2" target="_blank">CookieKeeperN2</a>
			<div class="markdown"><p>yup.  thanks</p></div>		</li>
					</ul>
		</ul>
		</ul>
	