	<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/Mingablo" target="_blank">Mingablo</a>
			<div class="markdown"><p>We did go through a bunch of technological leaps though. The HGP was done with Sanger sequencing and the nanopore and similar we've got today are hundreds of times cheaper per base.</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/available_username2" target="_blank">available_username2</a>
			<div class="markdown"><p>I know it's not $2.7 billion, but what's the cost of a oxford nanopore whole genome?</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/chicks_for_dinner" target="_blank">chicks_for_dinner</a>
			<div class="markdown"><p>This is a tough question because of the throughput of nanopore devices. Nanopores do long read sequencing and have a higher error rate and can’t sequence an entire human genome with enough “coverage”. Coverage comes from many sequenced reads aligning to the same locus in the genome. Illumina (short-read) sequencing can sequence fucktons of short fragments with many redundant reads for each base in the human reference sequence. This reduces technical errors. Nanopores and other long-read sequencing tech can get really long reads which is good for regions that are hard to map (think 10s of 1000s of repetitive sequences) but lack in coverage since fewer overall reads come from these experiments. This, of course, is rapidly evolving. I hope this helps.</p>
<p>Edit: short-reads = ~250 nucleotides per fragment, long-reads = ~10k-20k (Oxford Nanopore claims up to 250k with latest tech) nucleotides per fragment</p>
<p>So it really depends on the questions you want to answer with sequencing. For a lot of human health related stuff, Illumina (short-reads) is the standard, but people are now combining these analyses with long-read sequencing to look at larger, structural events which is an exciting newer field in genomics.</p>
<p>Another important note is that these experiments often look at germline mutations which are mutations acquired from your parents in addition to new mutations unique to you, which will then get passed on to the next generation. This gives little information on somatic mutation which are the mutations your cells accumulate from replication errors, mutagens, etc. that are not hereditary. These mutations can also be sequenced using the aforementioned technology, but prepared from a tissue sample rather than from blood DNA. You can also sequence RNA from individual cells this way. Really cool and complicated stuff (source: getting a PhD in computational genomics, so definitely don't quote me on any of this).</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/omgu8mynewt" target="_blank">omgu8mynewt</a>
			<div class="markdown"><p>Depends what coverage and accuracy you want. Nanopore is much less accurate than Illumina, so it depends how much you mind accidentally getting or missing SNPs. I tried to work it out mathematically, but got stuck. <a href="https://www.nature.com/articles/s41467-019-09637-5" target="_blank">This paper</a> was trying to work out what is the minimum amount of sequencing you need to successfully identify if someone has mutant proteins causing some diseases as a test. They used 73 flow cells at <a href="https://store.nanoporetech.com/uk/flowcells.html" target="_blank">about £400 each</a> if you're willing to spend £20,000 on bulk buying. You have to prepare the DNA before sequencing as well by library prep and purification and size selection, probably costs the same again. They got 80X average coverage, which seems reasonable. They also have professional bioinformaticians creating new base calling algorithms specifically for this project.</p>
<p>Their summary: </p>
<ul>
<li>Nanopore can't sequence mutant genes accurately enough for diagnostics</li>
<li>Data analysis algorithms are very much a work in progress </li>
<li>Very high computational requirements currently</li>
</ul>
<p>So my quick maths says about £58400 / $63617.35 for materials, then whatever you pay your scientists and also buy a supercomputer to run your analysis. Could be completely wrong, I normally sequence bacteria not people.</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/two_cat_morty" target="_blank">two_cat_morty</a>
			<div class="markdown"><p>I just worked on assembling a rodent genome, though we didn't use nanopore. We used the tech that is in between Sanger and nanopore (because nanopore is still a little too inaccurate for our purposes). It cost about $50k. So, a LOT cheaper than the original human genome but still a good deal more than $1000. However, now that we have the reference genome we can sequence individual genomes from that species or a closely related one for about $1000.</p></div>		</li>
					</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/pandizlle" target="_blank">pandizlle</a>
			<div class="markdown"><p>Sanger sequencing is still the golden standard. It’s just too damn accurate. Fortunately, the technology for automating sequencing now is better than having people read the gels by hand and eye.</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/__neone" target="_blank">__neone</a>
			<div class="markdown"><p>Ah memories. This was my job in college. I could get through analyzing and setting up the runs for 4 sequencers in less than an hour.</p></div>		</li>
					</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/BionicTransWomyn" target="_blank">BionicTransWomyn</a>
			<div class="markdown"><p>It kind of reminds me of the same discussion about space colonisation. Its a real prospect that should we ever be able to send a colony or exploration ship, long before it reaches its destination, we would already have the tech to build a new one that could get there before the first.</p>
<p>But if you follow that logic, we'd never launch any spaceships at all.</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/stonhinge" target="_blank">stonhinge</a>
			<div class="markdown"><p>The best option in such a scenario is just to go anyways, but leave funds and managers behind so that <em>when</em> technology improves you can send updated tech to be there waiting for the original colonists.</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/randomnumber42424242" target="_blank">randomnumber42424242</a>
			<div class="markdown"><p>I think there have been calculation for such wait times. If the celestial object cannot be reached within 50 years (i think it's 50, don't quote me), then it would be preferable to wait for developments in propulsion systems. Because the with new propulsion system you will reach the object before the older ships reach them. This is considering that the propulsion systems develop in current rate. I think there are also calculations that shows the existence of minimum total time per destination from the present moment. So we can calculate and logically decide what destination we can realistically reach with our present propulsion system.</p>
<p>edit:I read these ideas in some article and I don't remember any technical terms. So can't google them. I would be grateful if someone could give those terms.</p>
<p>edit 2: found this <a href="https://space.stackexchange.com/questions/25654/research-paper-on-wait-calculation-for-interstellar-travel" target="_blank">discussion</a></p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/naveed4367" target="_blank">naveed4367</a>
			<div class="markdown"><p>Not really, when it gets to the point where you can get there in  a a reasonable amount of time, then go for it.  Eventually you get to where there are only incremental gains in the technology and speed.</p></div>		</li>
					</ul>
		</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/Xenton" target="_blank">Xenton</a>
			<div class="markdown"><p>To add on to this, &quot;big jumble&quot; is surprisingly accurate way of describing it.</p>
<p>A huge part of the human genome project was getting fragments like:</p>
<blockquote>
<p>VER OTJ HE UMPSOV LOG EGOA JUM  OG THEG ERTH  TJUM HEL   RTHEL HEGO SOVER THELO</p>
</blockquote>
<p>and then piecing the overlapping parts of those fragments together to try and guess what the sentence is.</p>
<p>Now with a little guess work, you can create the sentence:</p>
<blockquote>
<p>THEGOATJUMPSOVERTHELOG</p>
</blockquote>
<p>But what made the human genome project so difficult is that, in this analogy: you're trying to piece that sentence together.... While trying to learn English.</p>
<p>Not only were they trying to turn the snippets into a coherent sentence, they didn't really know what a coherent sentence even looked like, yet.</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/Esslemut" target="_blank">Esslemut</a>
			<div class="markdown"><p>very well explained, thank you</p></div>		</li>
					</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/Dr-Mohannad" target="_blank">Dr-Mohannad</a>
			<div class="markdown"><p>So the reference genome was taken from whom to be made the standard we compare to or was it made my mapping a lot of genomes to account for mutations and abnormalities?</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/maestro2005" target="_blank">maestro2005</a>
			<div class="markdown"><p>The first reference genome was built from a pool of donors, and it ended up being spliced together from 13 of them, with one of them making up most of it.</p>
<p>It's important to remember that the HRG is not supposed to be a &quot;perfect&quot; person, merely one free of genetic abnormalities.</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/CheesecakeNo" target="_blank">CheesecakeNo</a>
			<div class="markdown"><blockquote>
<p>One free of genetic abnormalities</p>
</blockquote>
<p>You're right, but this is one of those &quot;by definition&quot; things, i.e. it's free of abnormalities because we defined the HRG as &quot;normal&quot;. When you map someone's sequence you say things like &quot;person X has a mutation at [location]&quot; the implied context is &quot;with respect to the reference genome&quot;. There's been an effort recently to redo the HRG, or create multiple of them for different ethnicities and populations to reduce this bias.</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/autoantinatalist" target="_blank">autoantinatalist</a>
			<div class="markdown"><p>How did they decide what was or wasn't abnormal? Like with lactose intolerance or allergies</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/slimejumper" target="_blank">slimejumper</a>
			<div class="markdown"><p>Craig Venter and James Watson ended up in the donor pool, what are the odds! /s</p></div>		</li>
					</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/accousticabberation" target="_blank">accousticabberation</a>
			<div class="markdown"><p>IIRC, it was a blend of different individuals from varied backgrounds,with multiple steps taken to ensure anonymity.</p>
<p><a href="https://www.genome.gov/human-genome-project/Completion-FAQ" target="_blank">https://www.genome.gov/human-genome-project/Completion-FAQ</a> has additional info about it.</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/_galaga_" target="_blank">_galaga_</a>
			<div class="markdown"><p>Long story, but in brief the initial government-funded effort was based on sequencing a minimal tile path of cloned DNA segments.  These segments came from libraries that often originated from a single individual, but multiple libraries were used, so the reference is a patchwork of human genomes.</p>
<p>I'm not sure where these projects are now, but there are efforts to produce a new human reference based on haploid human DNA from a single individual.  I just haven't kept up with that in a while.</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/amusing_trivials" target="_blank">amusing_trivials</a>
			<div class="markdown"><p>It was multiple libraries, but eventually the RP11 library became the majority.</p></div>		</li>
					</ul>
		</ul>
		</ul>
	