	<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/johnnydues" target="_blank">johnnydues</a>
			<div class="markdown"><p>Is there any fundamental problem with synthesizing longer DNA or is it mostly a problem of error rate or yield? I imagine that error rate is something that will improve a lot with time.</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/uthnara" target="_blank">uthnara</a>
			<div class="markdown"><p>It becomes exponentially harder to synthesize longer strands of DNA, you would actually make it in chunks and then stich it together which is the standard process for making longer sequences currently.</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/anonymous_teve" target="_blank">anonymous_teve</a>
			<div class="markdown"><p>There are many different methods to do it. Some suffer from error rate, some from yield.  But when you get down to it, one fundamental problem with very long stretches of DNA (chromosome-size) is how do you keep it from breaking? These are very thin filaments, and once they reach a certain size, they're not that stable. In our cells, we have a ton of proteins to take care of those huge strands of DNA. In a test tube, not so much.</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/setecordas" target="_blank">setecordas</a>
			<div class="markdown"><p>There are fundamental limitations in DNA synthesis, and especially RNA synthesis, that prevent the synthesis of very long stretches of nucleotides. With DNA, you reach a limit of around 200 to maybe 245 nucleotides. Each base has its own coupling efficiency (probability that it will couple to the next base in sequence), and the synthesizer, method, and reagents used can have a large impact as well. Overall coupling efficiency is back calculated once you've synthesized crude oligonucleotide by a simple equation: E = P^(n-1) where P is your average coupling efficiency and n is the length of the sequence. The solid support onwhich synthesis occurs normally has the first base attached, so you use n-1 in the exponent.</p>
<p>After cleavage and deprotection, you can run some of your crude sample on an LCMS to get a purity estimate of your full length product (FLP), and you can see how efficient or successful the synthesis was before you take it through purification. If you synthesized a 100nt strand of RNA and find 30% FLP purity through LCMS, then </p>
<p>0.3 = P^(99)</p>
<p>So your coupling efficience is 98.79% which is not great, but is not totally unexpected. If you are able to bump up your coupling efficiency to 99.5%, then you would expect a doubling of FLP to 60%, which is much, much better.</p>
<p>As you can see, small changes to coupling efficiency lead to substantial increases in purity. At the same time, increasing and decreasing strand length has a huge impact on yield. For RNA, an average coupling efficiency of 99.5% is very good.</p>
<p>If you took the method that gave you a 98.79% coupling efficiency and tried to synthesize a 200nt length oligo, you would have less than 9% crude FLP and 91% truncations and branchings. In comparison, a 99.5% average coupling efficiency would theoretically allow you to synthesize a 200nt oligo, but beyond around 150nt, coupling efficient will tend to drop off precipitously due to packing/stacking effects of the oligo on the solid support.</p>
<p>Techniques to get really long gene sequences involves enzymatically ligating short sequences until the desired length and sequence is aquired.</p></div>		</li>
					</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/adsfew" target="_blank">adsfew</a>
			<div class="markdown"><p>For the record, you can easily buy segments of DNA that are thousands of base pairs.</p>
<p>Source: Am a genetic engineer who has ordered these.</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/Kaalb" target="_blank">Kaalb</a>
			<div class="markdown"><p>Backing up your claim, my GF works in a production lab that makes Oligos up to thousands of base pairs in length lol. What a small world.</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/ECEXCURSION" target="_blank">ECEXCURSION</a>
			<div class="markdown"><p>Can you make dinosaurs like in Jurassic Park?</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/zekromNLR" target="_blank">zekromNLR</a>
			<div class="markdown"><p>Yep, those are made by making it in segments in the low hundreds of basepairs, that are then stuck together - which is why it is a problem to synthesise long sequences that have repeating patterns, because those repeating patterns can mess up the &quot;glue everything together in the correct order&quot; part.</p></div>		</li>
					</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/tinykeyboard" target="_blank">tinykeyboard</a>
			<div class="markdown"><p>^(not to mention epigenetics which play a large role in our development and everyday functioning. that information is not stored by these companies when they sequence your dna.)</p></div>		</li>
					</ul>
	