	<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/craftmacaro" target="_blank">craftmacaro</a>
			<div class="markdown"><p>To elaborate from the perspective of my field (I’m getting my PhD looking for medical applications and mechanisms of proteins I isolate from venoms I extract from our snakes) we get our raw materials for analysis from the natural source but just about all the drugs that have ever been developed from venom proteins (not antivenins, those are made from various products of the immune response produced in relatively large quantities in response to relatively small venom doses by larger mammals like sheep and horses so that isolating them from the animals is feasible for producing commercial amounts, I’m talking about drugs like Byetta, captopril (technically a chemically synthesized analogue of the peptide in Bothrops jararaca venom, small enough to not require biosynthesis), ziconotide, and many others that are currently approved or in various stages of research and clinical trials) are biosynthesized.  That was a hell of a sentence.</p>
<p>We are able to synthesize most snake venom proteins (snake venoms are rarely larger than 100 kd and most are significantly smaller... spider venoms are a different story) that we can sequence accurately by working backwards and figuring out a nucleotide sequence that produces that amino acid chain or from finding the mRNA transcript in a venom gland that is actively producing proteins (though this is a more recent method and still requires knowing which end amino acid sequence you’re looking for).  We then create a plasmid (basically a loop of DNA... think of it like a tiny chromosome with only a few genes that cells can share with one another through something called horizontal or lateral gene flow (they can spread it to other cells than those that result from their division) that contains that sequence along with those of a well understood method for controlling the activation of the plasmid like the lac operon and using a medium with lactose as well as a method for telling when your cells (bacteria, fungal, even insect... there tons of potential candidates) have received the plasmid by including the gene coding for an antibiotic resistance and using that antibiotic in your growth medium... only cells that have the plasmid and are actively transcribing and translating the proteins instructed by the plasmid sequence will express the resistance to the antibiotic (or the instructions to create an amino acid it might be deprived of in the growth media that the cell can’t normally create).  If it is able to survive in the conditions the plasmid helps with than we can assume that it’s likely also producing our venom protein or a fraction of the protein that we were interested in.  This allows us to make a lot more of the protein, as well as modify it, than we could get from extracting and purifying venoms.  It might also be a way to get a protein that’s difficult to purify from other proteins in the venom or one that the snakes produce as a dimer (2 separate amino acid chains connected by a non-peptide bond) which separation of would destroy the bioactivity of the protein when done using chromatography or chemical reduction methods.  </p>
<p>However there are also a ton of potential problems, including, but not limited to, the protein being irritating or harmful to the cells, in which case they often isolate the protein as an inclusion body stored in a special separate compartment of the cells at a pH or other condition which denatures (misfolds) the protein to remove the toxic or irritating nature.  This means when we isolate it the proteins are in a different structure than in venom and are no longer bioactive.  Sometimes we can use techniques that can refold them correctly, other times we can’t.  Their is also the possibility that to get an active form of the protein we might need the presence of a chaperone (a protein or other molecule whose presence at the time of synthesis changes the way the protein folds).  So there are lots of ways that make it difficult to mass produce a protein even when we have the amino acid sequence.  It’s even harder sometimes if we only have the genetic sequence since different cells can use different methods to determine what are exons (the parts of a transcribed mRNA sequence that go on to be translated into a protein) and which are introns (sections of the genetic code that are “cut” out before it is translated).  </p>
<p>TLDR: we use biosynthesis when we know the amino acid sequence of a protein and want to make a lot of it, such as when a venom protein has been determined to have medicinal properties and we need a lot more to medicate people than we can extract and isolate from snakes, jellyfish, snails, bees... take your pick, directly.  </p>
<p>I talk about it more from the perspective of someone working with proteins first and DNA/RNA second.  We can produce larger proteins far more accurately and with higher yields through the use of DNA coding for those amino acid chains than we can through chemical synthesis of proteins (which is about 99% accurate per amino acid (so 1% chance of a wrong amino acid being added every time), so with compounding error we can use chemical synthesis to get decent yields on proteins under 50 amino acids (50 chances for a wrong amino acid) but anything close to or above 100 amino acids and yields are extremely minimal).  However with biosynthesis we can get cells transfected with DNA instructions to produce proteins in that sequence with almost no limit to protein size (unlike when you are trying to produce lots of DNA that is all correct you only need to verify that one set of your DNA instructions are right and use that in your plasmid to be certain, barring a random mutation in the middle of your protein which is unlikely to happen before you have many thousands of accurate copies first, so the mutated one would be overwhelmed).  </p>
<p>TLDR Is TLDR:  Using DNA as a set of instructions to build long proteins is much more accurate than building them chemically.</p></div>		</li>
					</ul>
	