	<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/MimosasInMordor" target="_blank">MimosasInMordor</a>
			<div class="markdown"><p>Viruses are not my area of expertise, but in bacteria, there can be multiple sequences overlapping in opposite directions. One nucleotide could therefore be part of two separate reading frames. Plus, nobody actually sits down and manually reads the codons, we copy and paste the entire sequence (gaps or not) into a program that shows possible reading frames, and translates those possibilities into proteins. Visually separating a sequence into three for the benefit of a human reader would have little to no value. Gaps every 10 nucleotides just make it a little easier to count to a specific nucleotide, for example if a given sequence started on nucleotide 261, it's much easier to find this way than if there were no gaps at all.</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/mynameistoolongtosay" target="_blank">mynameistoolongtosay</a>
			<div class="markdown"><p>Codons aren’t intuitively identifiable just on a simple transcription. You need to identify the site where translation starts (which the program that prints these out doesn’t identify). Then you alone have to count codons three at a time from there. This is just the typical way to display this information.</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/WhoCaresWhyBother" target="_blank">WhoCaresWhyBother</a>
			<div class="markdown"><p>The virus genome can contain multiple open reading frames and so breaking it up into 3 would make it difficult to visualize the other frames. The sets of 10 bases are also standard for most sequence data.</p></div>		</li>
					</ul>
	