	<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/fermat1432" target="_blank">fermat1432</a>
			<div class="markdown"><p>H! So the virus is sequenced from bat tissue without isolating it?</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/iayork" target="_blank">iayork</a>
			<div class="markdown"><p>From bat swabs, not tissue, but yes. And that’s not interesting - that’s the routine way of doing things. The 20,000-some influenza genome sequences in a standard year are not isolates, they’re sequenced directly from swabs. Virus isolation isn’t quite a quaint, old-fashioned tool but it’s increasingly less important.</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/fermat1432" target="_blank">fermat1432</a>
			<div class="markdown"><p>Fascinating! Can you please direct me to a good intro to gene sequencing? Something for the layman would be great. Thanks!</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/avalon68" target="_blank">avalon68</a>
			<div class="markdown"><p>Its quite important for figuring out how the virus replicates to develop treatments. There was huge excitement when covid was cultured</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/fermat1432" target="_blank">fermat1432</a>
			<div class="markdown"><p>Do they make repeated measurements to ensure that all the genes are accounted for?</p></div>		</li>
					</ul>
		</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/keshav2191" target="_blank">keshav2191</a>
			<div class="markdown"><p>So research based on identified sequence happens in a simulated computer program? How does this work?</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/iayork" target="_blank">iayork</a>
			<div class="markdown"><p>No, that's not how it works. First, you can understand a huge amount of information from the sequence. You can understand the phylogeny (some segments of this influenza virus diverged from the main lineage about 400 years ago, but other segments diverged 1000 years ago and others split just 100 years ago, so you can infer how this virus interacted with the others). You can compare structural and functional findings (this influenza virus doesn't use the same receptor all the others do, so it must follow a different infection path). You can identify regions under positive and negative selection (here's how this virus adapted to bats). And much more.</p>
<p>But also see the part about infectious clone technology. You don't need to isolate a virus to make an experimental target. For example</p>
<blockquote>
<p>To identify cell lines susceptible to bat influenza virus, pseudotyped vesicular stomatitis virus (VSV) recombinants were used to identify susceptible mammalian cell lines [32,33,34]. The pseudotype viruses expressed the HA (VSVΔG-H17, VSVΔG -H18) of bat influenza virus instead of G protein and both were found to infect canine cells (MDCK II and RIE 1495) and human cells (A549, Calu-3, 16HBE14o, SK-Mel-28, U87 MG), but only the VSVΔG -H18 was able to infect the bat EpoNi/22.1 cells [32]. Another study revealed that the VSV pseudotyped with HA and NA of bat influenza virus entered bat cells (EpoNi/22.1, HypNi/1.1, and EidNi/41), but not cells from other mammalian species [34]. </p>
</blockquote>
<p>--<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8064322/" target="_blank">Bat Influenza Viruses: Current Status and Perspective</a></p>
<p>Depending on your definition of &quot;isolation&quot;, you can bypass that and still end up with a fully infectious virus infecting cultured cells, but starting with synthetic DNA instead of an &quot;isolate&quot; (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5111703/" target="_blank">Synthetically derived bat influenza A-like viruses reveal a cell type- but not species-specific tropism</a>).</p>
<p>Even more obviously, you don't need to isolate a virus to make proteins from it. For example, within a year of the first identification, the structures of some of the bat influenza proteins were published (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3503178/" target="_blank">Crystal structures of two subtype N10 neuraminidase-like proteins from bat influenza A viruses reveal a diverged putative active site</a>), which gives a lot of functional information.  </p>
<p>All this is hugely simplified, of course.</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/keshav2191" target="_blank">keshav2191</a>
			<div class="markdown"><p>Phew! So much to wrap my head around. But thanks for all the explanation. Will sit down and try and understand it!</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/sismetic" target="_blank">sismetic</a>
			<div class="markdown"><p>Hi! Thanks for taking the time to answer.</p>
<p>I am having trouble wrapping my head around this. Epistemologically, if you haven't isolated the virus, then how do you know that the object you're studying is, in fact, the one you're looking to study? How do you know it's not contaminated with similar but different material? How do you know it's not an entirely different material? Isn't isolation an epistemological necessity, especially when dealing with highly numerous and complex organisms like viruses?</p>
<p>I don't have much idea about it, but I can't wrap my head around as to why isolation is not a logical necessary step in order to be sure that your object of study is that which you want; maybe you could infer it from other points, but isn't that much less certain than isolating and eliminating the uncertainties?</p>
<p>Maybe it is a very dumb question, but how do you sequence a virus without isolating it? How do you get certainty that the sequence you've got is not contamined or of a similar yet different material? I understand some parts of the sequence may match certain expected parameters(like philology) but isn't that too &quot;in the dark&quot;, especially with possible combinations of DNA and pseudocoding(as I understand it), you would increase exponentially the probability of miscasting material, wouldn't it? Wouldn't the epistemological steps require you to reach certainty where one can, and hence require isolation?</p></div>		</li>
					</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/T_r0d" target="_blank">T_r0d</a>
			<div class="markdown"><p>I'll chime in here since i am currently doing a masters thesis in computational biochemistry, so i work with computer models simulating protein structures and interactions. More specifically, i am working on protein design, using these computational models to design new proteins that do not exist in nature. </p>
<p>There are certain types of research you can do simply with a sequence and computer simulations. The sequence of a gene carries information for the production of a protein, more specifically, the genetic nucleotide sequence is converted into the amino acid sequence for the resulting protein. The sequence of amino acids in turn decides the three-dimensional structure of that protein. The general idea is that a protein will adopt the three-dimensional structure that is most thermodynamically stable, and this is determined by the interactions of all amino acids. Charged amino acids may attract or repulse each other, small residues can come closer to each other than big ones, and so on. All of these interactions will contribute to the final energy of the protein. If we understand all of the forces acting on the protein as it is adopting its final structure, and if we know the amino acid sequence - which we will if we have the gene sequence - we should be able to predict the final structure of the protein. The structure of a protein is important to know since the structure determines the final function.</p>
<p>We are getting pretty good at performing structure predictions based only on the amino acid (or, genetic) sequence for a protein. We are getting to a point where these predictions are accurate enough for real-world applications. For a recent example, the Google-developed protein structure prediction program AlphaFold - the currently best such program - was used to give early predicted structures of the Sars-CoV-2 spike-protein and other associated viral proteins that had not been well studied or had their structure determined at the time. These predictions were based on the result from genetic sequencing. You can read about this here and see some protein models and how well they correspond to the actual structure, as determined by experimental methods: <a href="https://deepmind.com/research/open-source/computational-predictions-of-protein-structures-associated-with-COVID-19" target="_blank"><a href="https://deepmind.com/research/open-source/computational-predictions-of-protein-structures-associated-with-COVID-19" target="_blank">https://deepmind.com/research/open-source/computational-predictions-of-protein-structures-associated-with-COVID-19</a></a></p>
<p>The hope is for these tools to get so accurate that we can automatically identify proteins from genome sequences, complete with predicted structure and hopefully function. These tools are also used for the &quot;inverse&quot; problem, that is designing new proteins. The end goal there is to design proteins with specific functions, But, we are still far away from all this.</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/keshav2191" target="_blank">keshav2191</a>
			<div class="markdown"><p>Very clear and informative and with source. Thanks a bunch, man. Met so many good redditors today!</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/nikkikar" target="_blank">nikkikar</a>
			<div class="markdown"><p>This is very interesting. So what is a protein really and is there infinite? I know its basic but Please.</p></div>		</li>
					</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/ttk12acd" target="_blank">ttk12acd</a>
			<div class="markdown"><p>I think a more simple way to put it is that the sequencer sequence everything in the sample.  People can sort out what is in it afterward.  Say a sample has two things, if you know one you can figure out the rest is from another.  It get more difficult the more complex the sample is but the idea is the same.</p></div>		</li>
					</ul>
		</ul>
	