<div class="answers">
	<div class="answer" data-handle="frro72w">
		<a class="author" href="https://www.reddit.com/user/mix_feedback_repeat" target="_blank">mix_feedback_repeat</a>
		<span class="qa" title="Answer">A:</span><div class="markdown"><p>This question is a fundamental challenge to genetics. Hence, the answer is very complicated but I will try to give a basic explanation that might spur further additional questions.</p>
<p>The way I answer this is to explain that genes are turned on by proteins call “transcription factors,” which are themselves the products of genes that were turned on by transcription factors. You have a bunch of transcription factors floating around in the nucleus, but how do they know which genes to turn on? Well, they recognize and physically interact with DNA sequences that are (usually) somewhere in front (or “upstream”) of the target gene. These sequences are called “regulatory elements” and do not actually code a protein, so they are not considered genes. But don’t call them junk! Controlling elements are extremely important and are the source of a large portion of differences in traits between species. For instance, there might be a controlling element that tells a gene to only turn on the liver and to stay off in all other cells. When the transcription factors bind the controlling element, they can help “turn on” the gene, which means transcribing the DNA sequence into RNA, which then can be turned into a protein.</p>
<p>So, my extremely short answer to your question is transcription factors are turned on at a certain time and recognize their target genes by the non-coding DNA sequence near the target gene.</p>
<p>Tl;dr There are transcription factors and controlling elements that tell a gene when, where, and how much to turn a gene on and off.</p></div>		<div class="replies-controls">
			<a class="show-replies" href="javascript:void(0)">show replies...</a>
			<a class="hide-replies" href="javascript:void(0)">hide replies...</a>
		</div>
		<div class="replies-placeholder"></div>
	</div>
	<a class="less-answers upper" href="javascript:void(0)">less answers...</a>
	<div class="answer" data-handle="frsguaf">
		<a class="author" href="https://www.reddit.com/user/fluffyrhinos" target="_blank">fluffyrhinos</a>
		<span class="qa" title="Answer">A:</span><div class="markdown"><p>This is probably more detail than you want, but this is what I study, and I'm really interested in it, so I'd like to add some additional details to the other poster's answers. Transcription factors are responsible for turning genes on and off given the situations a cell encounters. But given that transcription factors are themselves genes, it creates a little bit of a chicken and egg problem; how do the transcription factors know when and how to turn on?</p>
<p>The other problem that cells must address is the sheer multitude of events and combinations of signals they can encounter. I specifically work with immune cells, and it is vitally important that they turn on genes in the correct place at the correct time. Any deviations can cause inflammatory disorders or allow infections/cancer to persist. But there aren't enough transcription factors in the cell to have a one-to-one connection for every possible scenario. There simply isn't enough space in the genome or physical space in the cell for all the proteins that would be needed.</p>
<p>To get around this, transcription factors respond to multiple signals, and signals often activate multiple transcription factors. This, in turn, raises the question of how the cell can differentiate signals that act on the same transcription factor. This question has many answers, but the one I am most interested in is dynamics. That is, the pathways that turn the transcription factors on, turn them on with a different temporal (and at times, spatial) pattern. This allows the cell to encode much more information than just looking at if the transcription factor is on or off (quantified in <a href="https://science.sciencemag.org/content/334/6054/354" target="_blank">this</a> and other papers).</p>
<p>I'll start with a really cool recent paper that shows how these dynamics can potentially be used therapeutically, and then I'll share more basic science papers (still equally cool!) below. This paper created a synthetic signaling pathway inside cells. It basically serves to detect the dynamics of proliferation signaling pathways inside that cell, and if those dynamics indicate hyperactivity (meaning the cell is likely canerous), then it triggers the cell to die. If the dynamics are normal, the cell is allowed to live (<a href="https://science.sciencemag.org/content/364/6439/eaat6982" target="_blank">source</a>). </p>
<p>This obviously a long, long way from the clinic, but a technology like this is exciting because it is harder for cancer cells to develop mechanisms to escape it. The hyperactivity of the growth signaling pathways is what causes the cancer cell to grow; it needs those signals to keep growing. So by stealing these signals, and making them kill the cell instead, it hits the cancer cell closer to the actual mechanism causing disease.</p>
<p>More to the basic science, and examples of this throughout biology, the classical example of transcription factor dynamics is seen in how neuronal precursor cells differentiate epidermal growth factor (EGF), which tells them to grow and divide, and neuronal growth factor (NGF), which tells them to turn into neurons. Both of these factors activate the same pathway (the ERK pathway), but EGF creates a pulse of activation, where NGF creates a sustained activation. This difference in dynamics turns on different genes and allows the cell to respond appropriately. This has been shown in many papers, but for <a href="https://www.sciencedirect.com/science/article/pii/0092867495904018" target="_blank">example</a>.</p>
<p>Another example is the p53 transcription factor, which is important in determining if a cell will survive, and therefore crucial in cancer. Too much p53 activation leads to apoptosis. For a long time, it was thought that if p53 activation reaches a certain level, the cell will die. However, as it turns out, the level of p53 a cell needs to reach is a dynamic target, that increases over time. In order for a cell to undergo apoptosis, p53 needs to activate both strongly and quickly to reach that target (<a href="https://pubmed.ncbi.nlm.nih.gov/27062928/" target="_blank">source</a>). </p>
<p>Finally, it was recently shown that immune cells use these dynamics to determine the danger of a bacterial infection. For example, having just a small number of bacteria around is significantly different than a cell which has bacteria growing inside it. This, in turn, requires the cell to respond differently. The cell encodes the level of the threat in the dynamics of proteins known as NF-kB and JNK (<a href="https://pubmed.ncbi.nlm.nih.gov/30904375/" target="_blank">source</a>). The fact that the cell uses dynamics, allows it to use these two proteins to respond to a whole host of possible scenarios.</p></div>		<div class="replies-controls">
			<a class="show-replies" href="javascript:void(0)">show replies...</a>
			<a class="hide-replies" href="javascript:void(0)">hide replies...</a>
		</div>
		<div class="replies-placeholder"></div>
	</div>
	<div class="answer" data-handle="frse8bs">
		<a class="author" href="https://www.reddit.com/user/flyover_liberal" target="_blank">flyover_liberal</a>
		<span class="qa" title="Answer">A:</span><div class="markdown"><p>Adding on to mix_feedback_repeat's answer:</p>
<p>Some transcription factors are hanging around in the cell waiting to be activated by an external stimulus.  Growth hormones bind to receptors and start a cascade of events that has transcription factors binding in a bunch of different places through the genome and making the transcription of those genes more likely.  </p>
<p>The external stimuli can be growth factors, other hormones, extracellular matrix proteins, immune signals, certain chemicals or reactive oxygen species, etc.  </p>
<p>There is also a mechanism to deactivate transcription factors in a lot of cases - sometimes this happens via chemical entropy, but fairly frequently there is an enzyme that catalyzes the reverse reaction that activated it in the first place.  </p>
<p>I'll cite NF-kB as an example (NF-kappaB).  It is comprised of two rel proteins (a dimer of the five rel proteins - last I checked, it's been a while since I checked in on this particular transcription factor).  It hangs out in the cytoplasm of the cell because it is hooked to an inhibitor protein, IkB.  IkB kinase, when it is activated from upstream, tags IkB with a phosphate group, which causes it to disassociate and be degraded.  The freed NF-kB dimer then translocates to the nucleus and binds to certain sequences of DNA.  This sequence is upstream of a bunch of inflammatory response proteins and other detoxification enzymes.</p></div>		<div class="replies-placeholder"></div>
	</div>
	<div class="answer" data-handle="frsi7ar">
		<a class="author" href="https://www.reddit.com/user/CocktailChemist" target="_blank">CocktailChemist</a>
		<span class="qa" title="Answer">A:</span><div class="markdown"><p>I can give an example from the field I studied in grad school. Thyroid hormones (TH) activate genes through the thyroid hormone receptor (TR) (most of this will also be broadly applicable for other nuclear hormone/receptor pairs, e.g. androgen, estrogen, vitamin A and D). The TR has two domains linked together - a DNA binding domain and a ligand binding domain. TH binding causes conformational changes in the receptor that alter its activity to recruit either coactivators or corepressors (proteins that influence the likelihood of RNA polymerase binding).</p>
<p>&#x200B;</p>
<p>The receptors are almost always bound to stretches of DNA called thyroid hormone response elements (TRE), which are usually within the promoter (where the RNA polymerase binds to initiate transcription) or enhancer (where other cofactors bind to increase the likelihood of initiation) regions upstream of the target gene. Those TREs can vary in form, function and strength - some will strongly induce target genes, others are weaker, and some will repress target genes rather than inducing them.</p>
<p>&#x200B;</p>
<p>This can get extra complicated when we introduce synthetic ligands into the mix, because they can alter TR conformation in ways that aren't available to TH. For instance, the antagonist NH-3 blocks both coactivator and corepressor recruitment.</p></div>		<div class="replies-placeholder"></div>
	</div>
	<div class="answer" data-handle="frun85q">
		<a class="author" href="https://www.reddit.com/user/3rdandLong16" target="_blank">3rdandLong16</a>
		<span class="qa" title="Answer">A:</span><div class="markdown"><p>Epigenetics. There are certain proteins whose job it is to activate and de-activate certain genes at specific points in development. A gene that does one thing in one stage of development might do something else entirely in another stage. This is called pleiotropy.</p>
<p>One common mechanism by which this occurs is via installation of a methyl group (i.e. methylation). This &quot;mutes&quot; a gene.</p></div>		<div class="replies-placeholder"></div>
	</div>
</div>