<div class="answers">
	<div class="answer" data-handle="gxqagrz">
		<a class="author" href="https://www.reddit.com/user/younghoon13" target="_blank">younghoon13</a>
		<span class="qa" title="Answer">A:</span><div class="markdown"><p>A virus we know exists, but we can't isolate in the lab is Molluscum contagiosum. It's a pox virus that we cannot culture in the lab, and to this day we don't have a good answer as to why. </p>
<p>There isn't alot of interest in studying it because it's benign and non lethal. However what makes it unique is that you can have what look like uninflamed smallpox like lesions for months to years.</p></div>		<div class="replies-controls">
			<a class="show-replies" href="javascript:void(0)">show replies...</a>
			<a class="hide-replies" href="javascript:void(0)">hide replies...</a>
		</div>
		<div class="replies-placeholder"></div>
	</div>
	<a class="less-answers upper" href="javascript:void(0)">less answers...</a>
	<div class="answer" data-handle="gxpwl4d">
		<a class="author" href="https://www.reddit.com/user/iayork" target="_blank">iayork</a>
		<span class="qa" title="Answer">A:</span><div class="markdown"><p>It depends on your definitions of &quot;know exists&quot; and &quot;isolate&quot;. To a large extent, advances in genetic sequencing have made traditional &quot;isolations&quot; obsolete. A huge amount of modern virology is done without &quot;isolated&quot; viruses, using sequence as the basis. It's very common to identify a virus from sequence and never bother trying to &quot;isolate&quot; it; or to use alternate molecular approaches (synthetic DNA cloning, protein expression, etc) to study the virus.</p>
<p>That's not to say that isolating a virus (which probably means finding a cell type it will grow on in culture) is not helpful. It opens up a lot of experiments you can do on the virus. But it's much less useful and interesting than it used to be, when isolation was the only way to identify viruses; now, this kind of isolation is just another, fairly clumsy and insensitive, way to look at viruses. </p>
<p>A good example would be the bat influenza viruses. They were first identified in 2012 (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3306675/" target="_blank">A distinct lineage of influenza A virus from bats</a>), based on sequence information. There are probably 100 papers discussing these viruses, their infection patterns, their receptors, etc etc, but they still haven't been traditionally isolated:</p>
<blockquote>
<p>Two unusual H17N10 and H18N11 influenza viruses have been found in New World bats. Although neither of these viruses have been isolated, infectious clone technology has permitted significant progress to understand their biology, which include unique features compared to all other known influenza A viruses. </p>
</blockquote>
<p>--<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8064322/" target="_blank">Bat Influenza Viruses: Current Status and Perspective</a></p></div>		<div class="replies-controls">
			<a class="show-replies" href="javascript:void(0)">show replies...</a>
			<a class="hide-replies" href="javascript:void(0)">hide replies...</a>
		</div>
		<div class="replies-placeholder"></div>
	</div>
	<div class="answer" data-handle="gxpwee1">
		<a class="author" href="https://www.reddit.com/user/CAElite" target="_blank">CAElite</a>
		<span class="qa" title="Answer">A:</span><div class="markdown"><p>The mechanism that 'triggers' people with a genetic predisposition to type 1 diabetes to become diabetic has been theorised to be some sort of pathogen that triggers the immune response that creates type 1 (body kills its own pancreatic beta cells).</p>
<p>Source: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2570378/" target="_blank">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2570378/</a></p></div>		<div class="replies-controls">
			<a class="show-replies" href="javascript:void(0)">show replies...</a>
			<a class="hide-replies" href="javascript:void(0)">hide replies...</a>
		</div>
		<div class="replies-placeholder"></div>
	</div>
	<div class="answer" data-handle="gxqyz1v">
		<a class="author" href="https://www.reddit.com/user/ShadowController" target="_blank">ShadowController</a>
		<span class="qa" title="Answer">A:</span><div class="markdown"><p>During the early 20th century there was a \~10 year period where over a million people came down with an illness that caused them to become a &quot;walking shell&quot; with low arousal, who would eventually lose their bodily functions and die. Over 500,000 people died during the epidemic, and the pathogen has never been identified, but it's suspected that it may be a virus: <a href="https://en.wikipedia.org/wiki/Encephalitis_lethargica" target="_blank"><a href="https://en.wikipedia.org/wiki/Encephalitis\_lethargica" target="_blank">https://en.wikipedia.org/wiki/Encephalitis\_lethargica</a></a></p></div>		<div class="replies-placeholder"></div>
	</div>
	<div class="answer" data-handle="gxq6p63">
		<a class="author" href="https://www.reddit.com/user/Jemima_Snuggleduck" target="_blank">Jemima_Snuggleduck</a>
		<span class="qa" title="Answer">A:</span><div class="markdown"><p>I am surprised that the expression metagenomics has not popped up here.</p>
<p>But the answer is, yes. You can take samples (such as a wastewater sample) and analyse all the genomes in the samples (using sequencing), and when you find new genomes you can give them a name, and you can perform phylogenetic analyses to correctly classify them. However, there can be many thousands of different genomes in the samples analysed, and you have not directly observed the species of which you have sequenced the genome. These newly identified species are given the prefix candidatus (Ca.).</p>
<p>Once the candidatus species has been isolated and cultured, it's name can be upgraded to its final form, and the Ca. is dropped.</p></div>		<div class="replies-placeholder"></div>
	</div>
</div>